### ISSUE TITLE
Synctex file correction broken with pdflatex

### ISSUE THREAD

**Issue by matt-42** [kmorel/UseLATEX#22]
The synctex file correction expects relative paths in the original syntex file but
pdflatex actually write absolute paths. This ends up in a synctex file with latex file paths
still pointing to the cmake tmp files:

```
    message("Replacing relative with absolute paths.")
    string(REGEX REPLACE
      "(Input:[0-9]+:)([^/\n][^\n]*)"
      "\\1${LATEX_SOURCE_DIRECTORY}/\\2"
      synctex_data
      "${synctex_data}"
      )
```

Here is the version of pdflatex:
pdfTeX 3.14159265-2.6-1.40.16 (TeX Live 2015/Arch Linux)

**Comment by luca-penasa** [comment]
got into this error too... Any solution?

**Comment by kmorel** [comment]
OK. I think I have a fix on pull-request #36. Can you try the UseLATEX.cmake in the following link and see if that fixes the problem for you?

https://github.com/kmorel/UseLATEX/blob/4d5eb7899dc7b6718c8890b671c1ad636c7ac9a6/UseLATEX.cmake

**Comment by kmorel** [comment]
While moving to the new repository and reviewing old issues, I noticed that this fix was sitting around without being merged. I merged in the change in the new repository, so I am closing the issue. If there are further issues, open a new one in the new repo at https://gitlab.kitware.com/kmorel/UseLATEX.
